creedengophp.properties

```diff
@@ -0,0 +1,14 @@
+category=External Analysers
+description=Provide a list of static code analyzers (for PHP language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
+homepageUrl=https://github.com/green-code-initiative/creedengo-php
+publicVersions=2.0.0
+archivedVersions=
+
+defaults.mavenGroupId=org.green-code-initiative
+defaults.mavenArtifactId=creedengo-php-plugin
+
+2.0.0.description=Initial Marketplace publication of plugin after renaming from "ecocode-php"
+2.0.0.sqVersions=[9.9,10.6.0]
```

Comment:
This is a brand new release. Are you sure you don't want it to be usable in the latest versions?

Reply:
I will test our plugin with the last plugin I will update if OK.
thank you for feedback

Reply:
finally, it's ok for 10.7 version but not ok with 10.8 for now.
If you doesn't mind, I prefer to manage 10.7 and 10.8 compatibilities inside next version.
thank you.

Reply:
It's up to you.